docs/Cloud_integrations/Device_Integration_Center/overview.md

```diff
@@ -6,83 +6,153 @@ brokers etc. There are two kind of events:
+      * `timestamp` *(type: long)* - Time of occurrence of the event in milliseconds.
+      * `endpointName` *(type: string)* - The device endpoint name given at creation.
+      * `domainId` *(type: string)* - The entire ID of the domain in which the device is located.
+      * `domainName` *(type: string)* - The name of the last subdomain without slashes.
+      * `url` *(type: string)* - LwM2M path (Resource Id) for which changes are occurring.
+      * `value` *(type: string|number)* - The value of the changed LwM2M resource. 
+
+        !!! Note "Value format"
+            Although it can be a number, in the case of `JSON` content type it must be enclosed in quotation marks.
+
+Available variables differs from context where they are used. See [headers](#headers) and [body](#body) for detailed information.
 
 ### Headers
 
-Variables can be used in the `Header value` field (variables defined in `Header key` will not be considered). You can
-enter any header value e.g. some token. If you change type to `$` you will be allowed to easily select some variable.
-However, in `abc` mode, variables can also be used.
+Variables can be used in the `Header value` field. You can
+enter any header value e.g. some token. If you change type to `$` you will be allowed to easily select available variables.
+However, in `abc` mode, variables can also be used with the help of the `$` sign.
+
+!!! Note
+
+    Variables cannot be used in the header key. Thus, any `$` references defined in the `Header key` will not be considered.
 
 ![Custom Headers Dialog](./images/custom-headers-dialog.webp){ width=80% }
 
-For device lifecycle events, there are the following variables:
+=== "Available lifecycle variables"
 
-* `version` - version of the event (type: string)
-* `timestamp` - time in milliseconds (type: long)
-* `endpointName` - device endpoint name (type: string)
-* `domainId` - domain id of device (type: string)
-* `domainName` - name of the last subdomain (type: string)
-* `subtype` - type of device event (type: string)
+      For device lifecycle events, there are the following variables:
+    
+      * `version`
+      * `timestamp`
+      * `endpointName`
+      * `domainId`
+      * `domainName`
+      * `subtype`
 
-In device telemetry events, there are the following variables:
+=== "Available telemetry variables"
 
-* `version` - version of the event (type: string)
-* `timestamp` - time in milliseconds (type: long)
-* `endpointName` - device endpoint name (type: string)
-* `domainId` - domain id of device (type: string)
-* `domainName` - name of the last subdomain (type: string)
+      For device telemetry events, there are the following variables:
+    
+      * `version`
+      * `timestamp`
+      * `endpointName`
+      * `domainId`
+      * `domainName`
 
 ### Body
 
+As mentioned in [Templating](#templating) section, you can define how your payload looks like.
```

Comment:
As mentioned in the [Templating](#templating) section, you can adjust the view/appearance/look of your payload.
If you want to return to the default template, click **Return to default**.

(we do not say "how it looks like" - it's a calque from Polish, instead we say "what it looks like", tho I would recommend avoiding it - it sounds weird in a documentation.
if possible, I would also avoid adding the "button" word, if you provide the name of the button and put it in bold, then it should be enough :)

Reply:
Thx